web-usfm-parser/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "usfm-grammar-web",
-  "version": "3.0.0-alpha.9",
+  "version": "3.0.0-alpha.2",
```

Comment:
Is this intentional?

Reply:
Yes. It was. When doing the fixes for \s5 support related error handling bugs and ignoreerror, it was done on both node and web versions. 
Node was at alpha.8 at that time. So to keep it uniform from now on bumped a fews numbers here in web